cosmo_tester/test_suites/test_blueprints/nodecellar_test_autoheal.py

```diff
@@ -129,7 +129,6 @@ def post_autoheal_assertions(self, before_state, after_state):
     def modify_blueprint(self):
         with YamlPatcher(self.blueprint_yaml) as patch:
             patch.merge_obj('groups', self.AUTOHEAL_GROUP_YAML)
-        print self.blueprint_yaml
```

Comment:
It's a "cosmetic aside change" - a "debug line" unnecessarily left behind. This is an old test, unrelated to snaphots..